app/controllers/VATExemptionController.scala

```diff
@@ -63,7 +63,7 @@ class VATExemptionController @Inject()(mcc: MessagesControllerComponents,
         value =>
           sessionService.save[Boolean](VATExemptionId.toString, value).map { cacheMap =>
             if (value) {
```

Comment:
I think this should also only do the TM update if the exception/exemption FS is disabled